spec/models/chargeback_vm_spec.rb

```diff
@@ -750,21 +750,21 @@ def result_row_for_vm(vm)
 
           subject { Chargeback::RatesCache.new.get(consumption).first }
 
+          before(:each) do
```

Comment:
The `(:each)` is unnecessary, it is the default

Reply:
Done, thanks 👍